scripts/h5toobj.py

```diff
@@ -130,43 +112,6 @@ def redis_bulk_str(r_str, host, port):
     logger.debug("Bulk insert r_str of len %d completed: %s", len(r_str), retout)
 
 
-def open_rados(ceph_conf, ceph_pool):
-    cluster = rados.Rados(conffile=ceph_conf)
-    cluster.connect()
-    available_pools = cluster.list_pools()
-    if ceph_pool not in available_pools:
-        logger.error("Specified pool %s not available in this cluster (%s)",
-                     ceph_pool, available_pools)
-        sys.exit()
-    ioctx = cluster.open_ioctx(ceph_pool)
-    pool_stats = ioctx.get_stats()
-    logger.info("Connected to pool %s. Currently holds %d objects totalling %g GB",
-                ceph_pool, pool_stats['num_objects'],
-                pool_stats['num_bytes'] / 2 ** 30)
-    return ioctx
-
-
-def write_chunk_rados(ioctx, name, chunk):
-    data_fp = io.BytesIO(chunk.data)
-    data_str = data_fp.read()
-    ioctx.write_full(name, data_str)
-
-
-def open_s3(url):
-    session = botocore.session.get_session()
-    config = botocore.config.Config(max_pool_connections=200,
-                                    s3={'addressing_style': 'path'})
-    client = session.create_client(service_name='s3', endpoint_url=url,
-                                   config=config)
-    return client
-
-
-def write_chunk_s3(client, name, chunk):
-    bucket, key = name.split('/', 1)
-    data_fp = io.BytesIO(chunk.data)
-    client.put_object(Bucket=bucket, Key=key, Body=data_fp)
-
-
```

Comment:
Looks like you started writing a docstring for `generate_chunks` and forgot to fill it in? (Github won't let me comment on the line since it didn't change).

Reply:
Yes. This function is in motion somewhere else and then I'll bother.